examples/natmod/stm32wb55_transparent_vcp/Makefile

```diff
@@ -0,0 +1,46 @@
+# This module is intended for use on stm32wb55 microcontrollers
+# and shows how a native module can be used to extend hardware
+# functionality.
+
+# Note: `pip install pyelftools` is required to build native modules.
+
+# Location of top-level MicroPython directory
+MPY_DIR = ../../..
+
+# Name of module
+MOD = transparent_vcp
+
+# Source files (.c or .py)
+SRC = \
+ _stm32wb55_transparent.c \
+ stm32wb55_transparent_vcp.py \
+
+
+# $(MPY_DIR)/ports/stm32/rfcore.c
```

Comment:
remove unused lines

Reply:
I've tried to properly clean up all the code / comments now in the latest push thanks.